test/proxyservice/secretless-with-errors.yml

```diff
@@ -0,0 +1,38 @@
+version: 2
+services:
+  http-connector:
+    connector: generic_http
+    listenOn: tcp://0.0.0.0:8080
+    credentials:
+      jwtToken:
+        from: literal
+        get: some-value
+    config:
+      headers:
+        ClientID: "12345678"
+        ClientSecret: "{{ .jwtToken }}"
+      match:
+        - ^http
+
+  tcp-connector:
+    connector: pg
+    listenOn: tcp://0.0.0.0:111111
```

Comment:
Might help to have comments pointing out all of the errors (make them obvious), e.g.
```
# Invalid TCP port
```